acceptance-tests/src/main/java/io/blueocean/ath/LogEntryLogger.java

```diff
@@ -18,11 +20,31 @@
     private static final ObjectMapper jsonMapper = new ObjectMapper();
     private static final TypeReference<HashMap<String, Object>> typeRef = new TypeReference<HashMap<String, Object>>() {};
 
-    static void recordLogEntry(LogEntry entry) {
-        if (!logger.isInfoEnabled() || isSuperfluousLogEntry(entry)) {
+    static void recordLogEntries(List<LogEntry> entries) {
+        if (!logger.isInfoEnabled()) {
             return;
         }
 
+        if (entries.size() == 0) {
+            logger.info("nothing written to browser console");
+            return;
+        }
+
+        List<LogEntry> usefulEntries = entries.stream()
+            .filter(entry -> !isSuperfluousLogEntry(entry))
+            .collect(Collectors.toList());
+
+        if (usefulEntries.iterator().hasNext()) {
+            logger.info("browser console output below:");
+            for (LogEntry entry : usefulEntries) {
+                LogEntryLogger.recordLogEntry(entry);
+            }
+        } else {
+            logger.info(String.format("nothing useful written to browser console; %s entries were hidden", entries.size()));
+        }
+    }
```

Comment:
Overall these changes make the browser logging stuff a bit more explicit about when 1. nothing is logged, 2. nothing is logged but useless messages, and 3. there are actually useful messages.